### PR TITLE
Low hanging fruit (pages): about, impressum, veranstaltungen, rubriken

### PR DESCRIPTION
**Pull request by ovbm**
Switches pages to colorSchemKey="auto", that work out of the box with darkmode or just required minor changes. 
tested

Examples:
Screenshot 2020-11-18 at 08 52 10
Screenshot 2020-11-18 at 08 56 03
Screenshot 2020-11-18 at 08 55 53

### CODE REVIEW

File: components/About/Page.js [hunk 1]
```diff
@@ -197,12 +195,12 @@ const AboutPage = ({
           <Subheader2>Geschäftsbericht 2019–2020</Subheader2>
           <P {...styles.faqCta}>
             Zu Beginn{' '}
-            <a
+            <Editorial.A
               {...styles.link}
```
Comment (tpreusse) [inline components/About/Page.js:199]: `{...styles.link}` can and probably should be removed everywhere.

**Review by tpreusse — COMMENTED** [review]
General approval, one clean up request

File: components/About/Page.js (hunk 1 above)
Comment (ovbm) [reply components/About/Page.js:199]: jup, didn't catch that!

Comment (ovbm) [reply components/About/Page.js]: done

**Pull request merged**